.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
I'd recommend adding these to a global `.gitignore` — would rather keep 'em out of the repo.